collector/config.yaml

```diff
@@ -4,6 +4,14 @@ receivers:
       grpc:
       http:
 
+processors:
```

Comment:
We don't have to enable these processors by default.

Reply:
Thank you, that make sense! In this case we make them available because we put them in `default.go`, but it is up to users to add them to their specific `config.yaml` files before they publish the layer as available 😄 .